consumer-configuration/README.adoc

```diff
@@ -130,17 +131,17 @@ rhoas kafka consumer-group describe [flags]
 
 For a list of consumer group commands that you can use with the CLI, see the {base-url-cli}{rhoas-cli-ref-url}[_CLI command reference (rhoas)_^].
 
-If you're using Kafka bin scripts, you can use the `kafka-consumer-groups.sh` tool to provide the same information:
+If you're using Kafka scripts, you can use the `kafka-consumer-groups.sh` tool to provide the same information:
 [source,subs="+quotes,+attributes"]
 ----
-bin/kafka-consumer-groups.sh --bootstrap-server __<bootstrap_server>__ --describe --group my-group-id
+./kafka-consumer-groups.sh --bootstrap-server __<bootstrap_server>__ --describe --group my-consumer-group
```

Comment:
@craicoverflow - I wasn't able to use the kafka-consumer-groups.sh script to perform operations like `--list`. 
Getting a  _...failed due to org.apache.kafka.common.KafkaException: Failed to find brokers to send ListGroups
_ error

Reply:
I am getting the same error, I guess this should be raised to teh kafka instances team.

Reply:
was this resolved?

Reply:
@wtrocki Not as far as I'm aware. Did you raise an issue @rkpattnaik780?

Reply:
Checking expectations with Mike Edgar

Reply:
Hi @rkpattnaik780 - we were missing the `--command-config` flag in the commands. The scripts seem to work now. Even created a new consumer group with the consumer script. :+1: 

I've added a new sub-section that describes the setup needed for the Kafka scripts. And updated the example commands.